docs/developer/editors.md

```diff
@@ -0,0 +1,175 @@
+# Editors
+
+Atomist is about automating more of our own work, and that includes the work of writing code. Not all code, of course! But the repetitive parts of coding, or the code that you want to change across many projects.
```

Comment:
"But the repetitive parts of coding,..." is not a sentence. I would recommend combining it with the previous sentence. Just change the exclamation mark to a comma.

Reply:
how about a lowercase b